vpnglean/src/logger.rs

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::os::raw::c_char;
+use std::ffi::CString;
+
+use log::{Level, LevelFilter, Log, Metadata, Record};
+
+// Logger implementation to integrate the Rust logs with loghandler.cpp.
+//
+// This message handler should be initialized alogside the Qt message handler.
+pub struct Logger {
```

Comment:
We might consider using this logger also for the Signature crate, WDYT? I can file a ticket.